src/main/python/run_regression_clueweb09b.py

```diff
@@ -37,9 +39,23 @@ def ndcg_eval(qrels, run):
     return round(float(os.popen("eval/gdeval.pl {} {} | grep 'amean' | sed 's/.*amean.//'".format(qrels, run)).read().split(',')[0]), 4)
 
 if __name__ == "__main__":
-    call(index_cmd, shell=True)
+    parser = argparse.ArgumentParser(description='Run regression tests on ClueWeb09b.')
+    parser.add_argument('--index', dest='index', action='store_true', help='rebuild index from scratch')
+    parser.set_defaults(index=False)
```

Comment:
`store_true` means if we provide `--index` to the script it will be `true`, right?
Then `parser.set_defaults(index=False)` is redundant.

Reply:
Well, if the `--index` flag isn't set, we need to default to False, right?

Reply:
https://docs.python.org/3/library/argparse.html#action

```
import argparse
  

if __name__ == "__main__":
  parser = argparse.ArgumentParser(description='store_true test')
  parser.add_argument('--index', dest='index', action='store_true')

  args = parser.parse_args()
  print(args)
```
`$ python3 test.py` 
`Namespace(index=False)`

`$ python3 test.py --index`
`Namespace(index=True)`

Reply:
I stand corrected!